README.md
# Lost-Cities
Heavy Mod Version of Dragon Knight. 

- Over 75 New or Modified Mods
- 200 Plus Levels
- 53 Different Character types/Races
- Graphic Interfaces for all Town Square Functions
- 72 Different Graphics for Map Exploring
- 50 Plus Random Events
- 12 Advance Random Events
- Many Quests
- 6 Very Advance Quests
- Approx. 700 Items (Weapons, Shields, Boots, Gloves, Range Weapons, Pets, Armor, Magic Rings)
    For Defense and offensive use for your Character
- Betting Houses
- Besides the Gold Coins, you can also trade in Silver and Copper Coins

Lost Cities Readme

### INTRODUCTION ###
Lost Cities
Installation Instructions

### Lost Cities Forum ###

Find Version Downloads, Graphics and Mods.

https://michaelmccart.com/dkbb/

### DOWNLAOD INSTRUCTIONS ###
Lost Cities is To Large to post here. You can download the whole game at:

https://www.michaelmccart.com/downloads/lc3133.zip - Approx. 400 MBs

Install instructions and SQL File:

https://www.michaelmccart.com/downloads/lc3133-Install.zip  4KBs

### SYSTEM REQUIREMENTS ###
- PHP (4.1 and higher)
- MySQL
- zlib compression enabled on your server (optional)

### INSTALLATION INSTRUCTIONS ###
1. Create a new database for Lost Cities to use, if you don't already have one set up.
2. Edit 'config.php' to include the correct values for your database setup.
3. Upload the contents of the Lost Cities folder to your site.
4. Upload the Lost Cities SQL to your Lost Cities Database.
5. To login: Admin User: SirMichael     Admin Passwood:  lc3133
6. Enjoy the game.




## #################### ##

Original Dragon Knight
Readme

### INTRODUCTION ###
Firstly, I'd like to say thank you for playing my game. The Dragon Knight game engine is the 
result of several months of planning, coding and testing. The original idea was to create a 
web-based tribute to the NES game, Dragon Warrior. In its current iteration, only the underlying 
fighting system really resembles that game, as almost everything else in DK has been made bigger 
and better. But you should still recognize bits and pieces as stemming from Dragon Warrior and 
other RPGs of old.

This is the first game I've ever written, and it has definitely been a positive experience. It 
got difficult at times, admittedly, but it was still a lot of fun to write, and even more fun to 
play. And I hope to use this experience so that if I ever want to create another game it will be 
even better than this one.

Once again, thanks for playing!

Jamin Seven
Dragon Knight creator

### LICENSE INFORMATION ###
The game is released as open-source with some limitations: You may install the game on your server 
and run it however you wish. You may also modify the program code for your own personal use in any 
way you see fit. You may not, however, redistribute the program (modified or stock) in any manner. 
And you may not alter the copyright or powered by notices at the bottom of the game pages in any 
manner. Short of those few rules, you are given free reign to do what you wish.

Being a free product, no technical support is guaranteed - however, you are free to post on the 
official DK forums if you need help with something. Due to time restraints, I will not answer support 
requests via email, so don't even try. By downloading the game, you agree to the above-mentioned rules, 
and you also agree not to hold the author responsible for anything in the event that something goes 
wrong. You use this program entirely at your own risk.

### COPYRIGHT ###
The Dragon Knight game engine in its entirety (scripts, images, etc.) is copyright 2003-2006 by
renderse7en and Jamin Seven, with exceptions for the few instances where I got coding help/tips
from replies to the php manual. Such instances are noted in the scripts.


rot/history.txt
0.0.1 - 2010-01-04 - Dustin Bolton
	Forked from featured-images2 version 1.1.2
0.0.2 - 2010-01-04 - Dustin Bolton
	Added widget functionality, image groups, multiple widgets per page.
0.0.3 - 2010-01-09 - Dustin Bolton
	Added file utility library into package, fixed javascript collision on group page, added clearer instructions.
0.0.4 - 2010-01-12 - Chris Jean
	Started transition from Featured Images 2 to Rotating Images
0.0.5 - 2010-01-12 - Chris Jean
	Completed transition to Rotating Images
0.1.0 - 2010-01-12 - Chris Jean
	Series of bug fixes
	Ready for beta release
0.1.1 - 2010-01-12 - Chris Jean
	Moved jquery.cross-slide.js embed to the footer.
0.1.2 - 2010-01-13 - Chris Jean
	Moved jquery enqueue back to rendering logic.
0.1.3 - 2010-01-13 - Dustin Bolton
	Fixed reorder buttons not working.
	Renamed js/admin-featured-images.js to js/admin-rotating-images.js
0.1.4 - 2010-01-13 - Chris Jean and Dustin Bolton
	Fixed just uploaded images disappearing.
	Got the group listing to properly show width and height.
	Reordered elements on the group editor to a more logical arrangement.
	Fixed a problem where editing an image could cause it to move to the first group.
0.1.5 - 2010-01-13 - Chris Jean
	Removed background image that causes issues with padding.
0.1.6 - 2010-01-13 - Chris Jean
	Fixed issue with group id not being sent when a new image is saved.
0.1.7 - 2010-01-14 - Dustin Bolton
	Modified option storage location for enhanced compatability. Any settings from previous versions will be lost.
0.1.8 - 2010-01-14 - Dustin Bolton
	Fixed issue with unescaped characters in display.
0.1.9 - 2010-01-15 - Chris Jean
	Updated version numbers to match
0.1.10 - 2010-01-15 - Dustin Bolton
	Limited admin backend to run only when needed.
0.1.11 - 2010-01-15 - Dustin Bolton
	Fixed conflict with Featured Images in Flexx Theme.
0.1.12 - 2010-01-18 - Dustin Bolton
	Fixed random image display when fading disabled.
	Footer no longer required in theme for plugin to function properly.
	Misc. code cleanup.
0.1.13 - 2010-01-18 - Dustin Bolton
	Fixed URL not changing on a per image basis when there is no overlay.
	Updated cross-slide javascript to latest version.
	Added sliding effect option.
0.1.14 - 2010-01-19 - Dustin Bolton
	Forcing img padding to 0 to resolve some theme conflicts (ie. builder has img padding in widget areas) causing cutoff due to wrapper.
0.1.15 - 2010-01-19 - Chris Jean
	Fixed problem with widget not working properly
	Removed default images
	Disabled text overlay by default
	Removed a number of warnings with variable checking
0.1.16 - 2010-01-19 - Chris Jean
	Moved the print_scripts hook back to the wp_footer action
0.1.17 - 2010-01-19 - Chris Jean
	Fixing hooks problem
0.1.18 - 2010-01-20 - Dustin Bolton
	Added centering option
0.1.19 - 2010-01-20 - Dustin Bolton
	Minor changes.
0.1.20 - 2010-01-20 - Dustin Bolton
	Re-added default image array until related issues caused by removal solved.
0.1.21 - 2010-01-21 - Dustin Bolton
	Removed default image array.
	Empty array sorting error now detected.
0.1.22 - 2010-01-27 - Dustin Bolton
	Revision lost.
0.1.23 - 2010-01-27 - Dustin Bolton
	Revision lost.
0.1.24 - 2010-01-27 - Dustin Bolton
	Fixed issues caused when only one image is uploaded.
0.1.25 - 2010-01-25 - Dustin Bolton
	Added shortcode support [it-rotate group="NUMBER_HERE"]
0.1.26 - 2010-02-01 - Dustin Bolton
	Attempting to fix Illegal offset type in cache.php.
0.1.27 - 2010-02-02 - Dustin Bolton
	Fixed image link when only one image is in a group.
	Added custom font family support for text overlay.
0.1.28 - 2010-02-04 - Dustin Bolton
	Added SSL check for linking rotating image files.
0.1.29 - 2010-02-04 - Dustin Bolton
	Added double transparency option to support both fading in and out images at the same time to handle transparent images.
0.1.30 - 2010-02-08 - Dustin Bolton
	Added additional documentation for shortcode usage.
0.1.31 - 2010-02-09 - Dustin Bolton
	Added instance variable for callign via PHP.
0.1.32 - 2010-02-22 - Dustin Bolton
	Added PluginBuddy.com footer.

clan.php
<?php  
// Original Estonian Clan Mod made by tanel tähepõld --> doublet in DK forum
// Translated and some recoding by Larry McDonnell AKA: larou / Lord Antarius E-MAIL ME: Larry_McD@hotmail.com

function leave() { // You may kick yourself from the clan
global $userrow;
if(isset($_POST['yes'])) {

$page .= "<div align=center> You have been kicked from the clan!
<br><br><a href=\"index.php\" class=\"myButton2\">Town Square</a></div> ";

$query = doquery("SELECT * FROM {{table}} WHERE owner='".$userrow["name"]."' LIMIT 1", "clans");
while ($clanrow = mysql_fetch_array($query)) { 
$experience = $clanrow["experience"] - $userrow["level"]; }
$updatequery = doquery("UPDATE {{table}} SET experience='$experience' WHERE owner='".$userrow["name"]."' LIMIT 1", "clans");
$updateuserquery = doquery("UPDATE {{table}} SET clanid='0', memberstatus='0' WHERE id='".$userrow["name"]."' LIMIT 1", "users"); }

elseif(isset($_POST['no'])) {
$page .= <<<END
<meta http-equiv="refresh" content="2;URL=index.php">
END;

$page .="<br><br><div align=center> You will be directed to the Main Page</div> "; }
else { 

$page .= "<br><div align=center>  Do you want to leave from this clan?  <br /><br /> Are you sure?<br><br></div> ";

$page .="<br><div align=center><form action=index.php?do=leave method=post><input type=submit value=Yes name=yes class=myButton2>&nbsp;&nbsp;&nbsp;<input type=submit value=No name=no class=myButton2></form></div><br><br>"; } 
display( 

$page, "<br><div align=center>Leaving the Clan</div><br><br>"); }

function create() { // In order to create a clan you must be 10th level
global $userrow;

if($userrow["level"] < 10) { display(" <div align=center>You haven't reach the required level to join a Clan.<br />You need to be at least a level 10.<br />To gain access to create a clan.
<br /><br /><a href=\"index.php?do=clans\" class=\"myButton2\">Clan Hall</a>
<br><br><a href=\"index.php\" class=\"myButton2\">Town Square</a></div><br><br>", "Error"); }

elseif (isset($_POST['create'])) {
if($_POST['name'] == "") { display(" <div align=center><br><br>You need to enter your Clan Name!<br /><br /><a href=\"index.php?do=clans\" class=\"myButton2\">Back to Clan Hall</a></div><br /><br />", "Error"); }
else { 

$page.="<br><div align=center>Your clan has been successfully created.<br /><br /> <a href=\"index.php?do=clans\" class=\"myButton2\">Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div><br><br>";

$query = doquery("INSERT INTO {{table}} SET id='', name='".$_POST['name']."', logo='".$_POST['logo']."', owner='".$userrow["name"]."'", "clans");
//update user who have made a clan. clanleader --> clanleader, if clanleader is one, that means that you are the leader of this clan, and memberstatus --> memberstatus. if member status is 5 then in clan after your name there are displayed LEADER
$query2 = doquery("UPDATE {{table}} SET clanid='".$userrow["name"]."', clanleader='1', memberstatus='7' WHERE id='".$userrow["name"]."' LIMIT 1", "users"); } }
else { 

$page .= "<br><br><div align=center> <form action=index.php?do=create method=post><br /> ";
$page .= "Clans name:&nbsp;&nbsp;&nbsp;<input type=text name=name size=30 max=30><br /><br /> ";
$page .= "Clans logo:&nbsp;&nbsp;&nbsp;<input type=text name=logo size=30 max=30><br /><br>";
$page .= "<input type=submit value=Create name=create></form><br /><br /> ";
$page .= "Return to the <a href=\"index.php\" class=\"myButton2\">Town Square</a></div><br><br>"; }
display( 
$page, $title); }

function rank($id) { // Here you may change a members rank // boss, soldier, noobie ect.
global $userrow;   
if(isset($_POST['change'])) {
doquery("UPDATE {{table}} SET memberstatus='".$_POST['memberstatus']."' WHERE id='$id' LIMIT 1", "users");


$page .= "<center><h3 class=\"title\">Clan Members Rank Changed</h3></center><br /><br /><br /><br />";
$page .="<br><div align=center>Members rank has changed.<br /><br /><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div><br /><br />"; }
else { $query = doquery("SELECT * FROM {{table}} WHERE id='$id' LIMIT 1", "users");
while ($row = mysql_fetch_array($query)) { 
$member = $row["charname"];


$page .= "<center><h3 class=\"title\">Change Clan Members Rank</h3></center><br /><br /><br /><br />";
$page .= "<div align=center><form action=index.php?do=rank:$id method=post><br />";
$page .= "$member's rank: <input type=text name=memberstatus size=1>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;";
$page .= "<input type=submit value=Change name=change class='myButton2'></form><br /><br /><br />";


$page .= "Legend: 0 = Squire | 1 = Man-at-Arms | 2 = Bill-Man | 3 = Banneret Knight<br />4 = Bachelor Knight | 5 = Feudal Lord | 6 = Constable | 7 =Marshal";
$page .="<br><br /><br /><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div><br /><br />"; } }
display( 
$page, "<div align=center>Change Clan Members Ranks</div>"); }

function kick($id) { // To kick a member from your clan
global $userrow;
$updatequery  = doquery("UPDATE {{table}} SET clanid='0', memberstatus='0', usemine='0' WHERE id='".$id."' LIMIT 1", "users");
$query = doquery("SELECT * FROM {{table}} WHERE id='$id' LIMIT 1", "users");
while ($row = mysql_fetch_array($query)) { 
$clanquery = doquery("SELECT*FROM {{table}} WHERE owner='".$userrow["name"]."' LIMIT 1", "clans");
while ($clanrow = mysql_fetch_array($clanquery)) {

$k = $row["level"] * 2;
$newexperience = $clanrow["experience"] - $k;
$experienceupdate = doquery("UPDATE {{table}} SET experience='$newexperience' WHERE owner='".$userrow["name"]."' LIMIT 1", "clans"); } }

$page .= "<div align=center>Member is kicked from your clan!<br /><br /><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div><br><br>";
display( 
$page, "<br><div align=center>Member kicked</div><br>"); }

function yes($id) { // To accept a member in your clan
global $userrow;
$usersquery = doquery("SELECT*FROM {{table}} WHERE id='$id' LIMIT 1", "users"); //you get exp for you clan when member unites
while ($userrow = mysql_fetch_array($usersquery)) { 
$usersupdate = doquery("UPDATE {{table}} SET clanid='".$userrow["name"]."' WHERE id='".$userrow["name"]."'", "users");//member accepted
$clanquery = doquery("SELECT*FROM {{table}} WHERE owner='".$userrow["name"]."' LIMIT 1", "clans");
while ($clanrow = mysql_fetch_array($clanquery)) { 
$k = $userrow["experience"] * 2;
$newexperience1 = $clanrow["experience"] + $k; //new xp
$update = doquery("UPDATE {{table}} SET experience='$newexperience1' WHERE owner='".$userrow["name"]."' LIMIT 1", "clans");

$page .= "<br><div align=center>Members application was accepted <br /><br />Your clan has gained $k Experience. 
<br><br><br><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div> ";
$delete = doquery("DELETE FROM {{table}} WHERE id='$id'", "users"); } }
display( $page, "Application Accepted"); }

function no($id) { // If you dont accept member in to your clan
$page .= " <div align=center> Members application was deleted! <br /><br />";
$page .= " Return to your <a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div> ";
doquery("DELETE FROM {{table}} WHERE id='$id'", "users");
display( 
$page, " <div align=center> Application deleted </div> "); }

function application($id) { // To apply for a membership in a clan
global $userrow;

if($userrow["usemine"] == 1) { 
$page .= "<br/><br /><div align=center> You have already sent your application!<br/><br /><a href=\"index.php\" class=\"myButton2\">Town Square </a> </div> "; }
else { doquery("INSERT INTO {{table}} SET id='', clanid='$id', experience='".$userrow["level"]."', id='".$userrow["name"]."', username='".$userrow["charname"]."'", "users");
doquery("UPDATE {{table}} set usemine='1' WHERE id='".$userrow["name"]."'", "users");
$page .= " <div align=center> Your application was sent! <br /><br /> ";
$page .= "<a href=\"index.php?do=clans\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div> "; }
display( 
$page, " <div align=center> Application sent </div> "); }

function clans() { // Clans create // use
global $userrow;

if($userrow["clanid"] == 0) {
$page .= "<center><h3 class=\"title\">Create a Clan</h3></center><br /><br /> ";
$page .= "<div align=center><table cellpadding=\"0\" cellspacing=\"0\"><tr><td><a href=\"index.php?do=create\" class=\"myButton2\">Create A Clan?</a></td></tr></table></div><br/><br /> ";
$page .= "<div align=center><table cellpadding=\"0\" cellspacing=\"0\" style=\"border: solid 2px black\" width=\"70%\"> ";
$query = doquery("SELECT * FROM {{table}} ORDER BY experience DESC LIMIT 100", "clans");
$rank = 1;
while ($clanrow = mysql_fetch_array($query)) {
$page .= " <tr><td bgcolor=\"#ffffff\">&nbsp;&nbsp;$rank</td><td bgcolor=\"#ffffff\">&nbsp;&nbsp;".$clanrow["name"]."</a></td><td bgcolor=\"#ffffff\">&nbsp;&nbsp;Experience: ".$clanrow["experience"]."</td><td bgcolor=\"#ffffff\">&nbsp;&nbsp;<a href=\"index.php?do=use:".$clanrow["owner"]."\">Join</a></td></tr>\n";
$rank++; }
$page .= " </table>\n<br /><br /><br /><br />\n";
$page .= "<a href=\"index.php\" class=\"myButton2\">Town Square</a></div> ";
$title = " <div align=center>Join this clan</div> "; }

elseif($userrow["clanleader"] == 0 & $userrow["clanid"] > 0 ) { // Normal clan member
$clanquery = doquery("SELECT logo FROM {{table}} WHERE owner='".$userrow["clanid"]."' LIMIT 1", "clans");
while ($clanrow = mysql_fetch_array($clanquery)) {
$logo = $clanrow["logo"]; }
$page .= "<center><h3 class=\"title\">Clans</h3></center><br /><br /> ";
$page .= " <div align=center><img src=\"$logo\"> "; // Display clan logo
$page .= " <form action=index.php?do=clans method=post> ";
$page .= " <table><tr><td><input type=submit value='Clan Info' name=clan></td><td><input type=submit value='Messages' name=messages></td><td><input type=submit value='Members' name=members></td><td><input type=submit value='Main page' name=tagasi></td></tr><tr><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></td></tr><tr></tr><tr> &nbsp;&nbsp;&nbsp;&nbsp; <a href=\"index.php?do=leave\" Leave from this clan! </a></td></tr></table> </div> "; }

elseif ($userrow["clanleader"] == 1) { // clan leader
$clanquery = doquery("SELECT logo FROM {{table}} WHERE owner='".$userrow["name"]."' LIMIT 1", "clans");
while ($clanrow = mysql_fetch_array($clanquery)) {

$logo = $clanrow["logo"]; }
$page .= "<center><h3 class=\"title\">Clan Information</h3></center><br /><br /> ";
$page .= "<div align=center><table border=\"0\" bgcolor=\"#000000\" cellpadding=\"0\" cellspacing=\"0\" style=\"outline-style:outset;\"><tr><td><div align=center><img src=\"$logo\"/></td></tr></table>";
$page .= " <br /><br /><form action=index.php?do=clans method=post> ";
$page .= " <table><tr><td><br /><br /><input type=submit value='Clan Info' name=clan class='myButton2'></td><td><br /><br /><input type=submit value='Messages' name=messages class='myButton2'></td><td><br /><br /><input type=submit value='Members' name=members class='myButton2'></td><td><br /><br /><input type=submit value='Leader Area' name=admin class='myButton2'></td><td><br /><br /><input type=submit value='Main page' name=tagasi class='myButton2'></td></tr><tr></tr><tr><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></td></tr></table><br /><br /> </div> "; }

if(isset($_POST['clan'])) {
$clanquery = doquery("SELECT*FROM {{table}} WHERE owner='".$userrow["clanid"]."' LIMIT 1", "clans");
while ($clanrow = mysql_fetch_array($clanquery)) {
$ownerquery = doquery("SELECT*FROM {{table}} WHERE id='".$clanrow["owner"]."' LIMIT 1", "users");
while ($ownerrow = mysql_fetch_array($ownerquery)) {
$page .="<div align=center><table border=\"0\"  width=\"65%\"><tr><td>Clan name:   ".$clanrow["name"]."</td>";
$page .= "<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>";
$page .= "<td>Owner of Clan:   <a href=\"index.php?do=onlinechar:".$ownerrow["id"]."\">".$ownerrow["charname"]."</a></td>";
$page .= "<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>";
$page .= "<td>Clan fame:   ".$clanrow["experience"]."</td></tr></table></div> "; } } }

if(isset($_POST['messages'])) {
$clanquery = doquery("SELECT * FROM {{table}} WHERE owner='".$clanrow["clanid"]."' LIMIT 1", "clans");
$newsquery = doquery("SELECT * FROM {{table}} WHERE clanid='".$userrow["clanid"]."' ORDER BY id DESC LIMIT 5", "users");
$rank = 1;
while ($newsrow = mysql_fetch_array($newsquery)) { 
$page .="<div align=center><table border=\"0\"  width=\"65%\"><tr><td>Rank:  $rank.</td>";
$page .= "<td>Subject:   ".$newsrow["subject"]."</td>";
$page .= "<td>Member:   <a href=\"index.php?do=onlinechar:".$newsrow["id"]."\">".$newsrow["charname"]."</a></td></tr></table></div>";
$rank++; } }


elseif(isset($_POST['addmessage'])) {
$page .= "<div align=center><form action=index.php?do=clans method=post>";
$page .= "Message:<br /><textarea name=\"subject\" rows=\"6\" cols=\"50\" max=50></textarea><br /><br /><input type=submit name=submit value='Submit'  class='myButton2'></div>"; }

elseif(isset($_POST['submit'])) {
$usersquery = doquery("SELECT * FROM {{table}} WHERE clanid='".$userrow["clanid"]."' LIMIT 1", "users");
$usersquery = doquery("UPDATE {{table}} SET subject='".$_POST['subject']."'", "users");
$page .= " <div align=center> Your message was added </div> "; }

elseif(isset($_POST['admin'])) {
$page .= " <div align=center> <form action=index.php?do=clans method=post>";
$page .= "<table><tr><td><input type=submit value='Clan settings' name=settings class='myButton2'></td><td></td><td><input type=submit value='Add message' name=addmessage class='myButton2'></td><td><input type=submit value='Ranks' name=ranks class='myButton2'></td><td><input type=submit value='Kick' name=kick class='myButton2'></td><td><input type=submit value='Applications' name=applications class='myButton2'></td></td></tr><tr></tr></table> </div> "; }

elseif(isset($_POST['applications'])) {
$page .= "<div align=center><table><tr><td><input type=submit value='Add message' name=addmessage class='myButton2'></td><td><input type=submit value='Ranks' name=ranks class='myButton2'></td><td><input type=submit value='Kick' name=kick class='myButton2'></td></tr></table> ";
$page .= " <table width=\"80%\">";
$query = doquery("SELECT * FROM {{table}} WHERE clanid='".$userrow["name"]."' ORDER BY id DESC LIMIT 100", "users");
$rank = 1;
while ($row = mysql_fetch_array($query)) {
$clanquery = doquery("SELECT * FROM {{table}} WHERE id='".$row["userid"]."'", "users");
while ($clanrow = mysql_fetch_array($clanquery)) {
$page .= "<tr><td>Rank:   $rank</td><td>User Name:   <a href=\"index.php?do=onlinechar:".$row["userid"]."\" class=\"myButton2\">".$row["usersname"]."</a></td><td>Level:   ".$clanrow["level"]."<td width=\"100%\"><a href=\"index.php?do=yes:".$row["id"]."\" class=\"myButton2\">[Yes]</a></td><td width=\"100%\"><a href=\"index.php?do=no:".$row["id"]."\" class=\"myButton2\">[No]</a></td></tr>\n";
$rank++; } }
if ($row = mysql_fetch_array($query) == 0) { 
$page .= " <tr><td width=\"100%\"><div align=center>You dont have any applications.</div></td></tr>\n"; }
$page .= " </table>\n<br /><br />\n </div> "; }

elseif(isset($_POST['settings'])) {
$page .= "<center><h3>Clan Settings</h3></center><br />";
$page .= "<div align=center><form action=index.php?do=clans method=post>"; 
$page .= "Clans Name:   <input type=text name=clanname size=40>&nbsp;&nbsp;&nbsp;";
$page .= "<input type=submit value='Change' name=name class='myButton2'><br /><br />";
$page .= "<br /><br />Clans Logo Address:  ";
$page .= "<input type=text name=clanlogo size=40>&nbsp;&nbsp;&nbsp;";
$page .= "<input type=submit value='Change' name=logo class='myButton2'><br />";
$page .= "<br />Your Logo should be 400w X 150h pixels in size<br>Logos larger that this size will result in deletation of Clan.<br /> ";

$page .= "<br />Use: http://yourdomain.com/yourlogo.png or<br /> ";
$page .= "<br />Use one of our preset Clan Logos:<br /> ";
$page .= "<br />Enter: images/clans/1.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/1.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/2.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/2.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/3.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/3.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/4.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/4.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/5.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/5.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/6.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/6.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/7.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/7.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/8.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/8.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/9.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/9.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/10.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/10.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/11.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/11.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";
$page .= "<br />Enter: images/clans/12.png in 'Clans Logo Address' box.<br /> ";
$page .= "<br /><img src=\"images/clans/12.png\" width=\"400\" height=\"150\" alt=\"Clan Logo\" border=\"0\" style=\"outline-style:outset;\"><br /> ";

$page .= "<br /><br /><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div>"; }

elseif(isset($_POST['name'])) {
$page .= "<div align=center><br />Clans Name Has Changed!<br /><br /><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div>";
doquery("UPDATE {{table}} SET name='".$_POST['clanname']."' WHERE owner='".$userrow["name"]."' LIMIT 1", "clans"); }

elseif(isset($_POST['logo'])) {
$page .= " <div align=center> <br /><br />Clans Logo Has Changed!<br /><br /><a href=\"index.php?do=clans\" class=\"myButton2\">The Clan Hall</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href=\"index.php\" class=\"myButton2\">Town Square</a></div> ";
doquery("UPDATE {{table}} SET logo='".$_POST['clanlogo']."' WHERE owner='".$userrow["name"]."' LIMIT 1", "clans"); }

elseif(isset($_POST['ranks'])) {
$page .= "<div align=center> <form action=index.php?do=clans method=post>";
$page .= "<table><tr><td><input type=submit value='Add message' name=addmessage class='myButton2'></td><td><input type=submit value=Members name=members class='myButton2'></td><td><input type=submit value='Ranks' name=ranks class='myButton2'></td><td><input type=submit value=Kick name=kick class='myButton2'></td></td><td><input type=submit value='Main page' name=back class='myButton2'></td></tr><tr></tr></table><br /><br />";
$page .= "<table width=\"80%\">";
$query = doquery("SELECT * FROM {{table}} WHERE clanid='".$userrow["clanid"]."' ORDER BY memberstatus DESC LIMIT 1000", "users");
$rank = 1;

while ($row = mysql_fetch_array($query)) {
if($row["memberstatus"] == 0) { $status = "Squire"; } elseif($row["memberstatus"] == 1) { $status = "Man-at-Arms"; } elseif($row["memberstatus"] == 2) { $status = "Bill-Man"; } elseif($row["memberstatus"] == 3) { $status = "Banneret Knight"; } elseif($row["memberstatus"] == 4) { $status = "Bachelor Knight"; } elseif($row["memberstatus"] == 5) { $status = "Feudal Lord"; }  elseif($row["memberstatus"] == 6) { $status = "Constable"; }  elseif($row["memberstatus"] == 7) { $status = "Marshal"; }
$page .= "<tr><td>Rank:   $rank</td><td>Member:   <a href=\"index.php?do=onlinechar:".$row["id"]."\">".$row["charname"]."</a></td><td>Level:   ".$row["level"]."</td><td> Rank: </td><td>Status:   $status</td><td><a href=\"index.php?do=rank:".$row["id"]."\" class=\"myButton2\">Change Rank</a></td></tr>\n ";
$rank++; }
$page .= "</table>\n<br /><br />\n </div> "; }

elseif(isset($_POST['members'])) {
$page .= " <div align=center> <table width=\"65%\">";
$membersquery = doquery("SELECT * FROM {{table}} WHERE clanid='".$userrow["clanid"]."' ORDER BY memberstatus DESC LIMIT 1000", "users");
$rank = 1;

while ($row = mysql_fetch_array($membersquery)) {
if($row["memberstatus"] == 0) { $status = "Squire"; } elseif($row["memberstatus"] == 1) { $status = "Man-at-Arms"; } elseif($row["memberstatus"] == 2) { $status = "Bill-Man"; } elseif($row["memberstatus"] == 3) { $status = "Banneret Knight"; } elseif($row["memberstatus"] == 4) { $status = "Bachelor Knight"; } elseif($row["memberstatus"] == 5) { $status = "Feudal Lord"; }  elseif($row["memberstatus"] == 6) { $status = "Constable"; }  elseif($row["memberstatus"] == 7) { $status = "Marshal"; }
$page .= "<tr><td>Rank:   $rank</td><td>Member:   <a href=\"index.php?do=onlinechar:".$row["id"]."\">".$row["charname"]."</a></td><td>Level:   ".$row["level"]."</td><td>Rank:   $status</td></tr>\n";
$rank++; }
$page .= "</table>\n<br /><br />\n </div> "; }

elseif(isset($_POST['kick'])) {
$page .= " <div align=center> <form action=index.php?do=clans method=post>";
$page .= "<table width=\"80%\">";
$membersquery = doquery("SELECT * FROM {{table}} WHERE clanid='".$userrow["clanid"]."' ORDER BY memberstatus DESC LIMIT 1000", "users");
$rank = 1;
while ($membersrow = mysql_fetch_array($membersquery)) {

if($membersrow["clanleader"] == 1) {  $kick = "";  }
else { $kick =" <a href=\"index.php?do=kick:".$membersrow["id"]."\">[Kick member]</a>"; }
$page .= "<tr><td width=\"10%\">$rank</td><td width=\"50\"><a href=\"index.php?do=onlinechar:".$membersrow["id"]."\" class=\"myButton2\">".$membersrow["charname"]."</a></td></td><td>$status</td><td>$kick</td></tr>\n";
$rank++; }
$page .= "</table>\n<br /><br />\n </div> "; }
display( 
$page, "The Clans"); }

?>